### DISCUSSION TITLE
How to `go back` after `go to definition`

### DISCUSSION THREAD

**Discussion by nikivdev** [zed-industries/zed#6630] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the feature

This is essential binding for me but I can't find how to do it.

I searched here: https://zed.dev/ref/default.json

Thought it was `"shift-f8": "editor::GoToPrevDiagnostic",` but it was not.

What is it?

**Comment by bigspawn** [comment]
Hi! I think these examples could be helpful https://github.com/zed-industries/keymaps

It seems that what you're looking for is hiding behind there key bindings

```
      "ctrl--": "pane::GoBack",
      "ctrl-_": "pane::GoForward",
```

**Comment by nikivdev** [comment]
`pane::GoBack` is what I needed, thank you
